src/components/DetailJobPage/DetailJobPage.jsx

```diff
@@ -0,0 +1,218 @@
+import React, { Component } from "react";
+import {
+  FaSearchLocation,
+  FaPhone,
+  FaBook,
+  FaBookmark,
+  FaMailBulk,
+  FaMoneyBillAlt,
+  FaStopwatch,
+  FaTransgender,
+  FaMale,
+  FaEnvelope,
+  FaBookOpen
+} from "react-icons/fa";
+import { connect } from "react-redux";
+import { injectIntl, FormattedMessage } from "react-intl";
+import { updateJob } from "../../lib/redux/actions";
+
+class DetailJobPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      jobDetail: []
+    };
+  }
+
+  componentDidMount() {
+    this.props
+      .http("auth/detail/post/" + this.props.match.params.id)
```

Comment:
Không viết theo cách này e nhé

this.props.http({
url: auth/detail/post
params: {
post_id: this.props.match.params.id
}
})